include/rocksdb/table.h

```diff
@@ -100,6 +100,15 @@ struct BlockBasedTableOptions {
 
   IndexType index_type = kBinarySearch;
 
+  // The index type that will be used for the data block.
+  // Now two DataBlockIndexType supported
+  enum DataBlockIndexType : char {
+    kDataBlockBinarySearch = 0,  // traditional block type
+    kDataBlockHashSearch = 1,     // additional hash index appended to the end.
```

Comment:
Did you add an option that does nothing? Can you at least mention it in comment that it is not yet implemented?

Reply:
Just addressed in a new PR https://github.com/facebook/rocksdb/pull/4203